packages/backend-test-utils/src/next/wiring/TestBackend.ts

```diff
@@ -101,4 +127,30 @@ export async function startTestBackend<
   }
 
   await backend.start();
+
+  return backend;
 }
+
+let registered = false;
+function registerTestHooks() {
+  if (typeof afterEach !== 'function') {
+    return;
+  }
+  if (registered) {
+    return;
+  }
+  registered = true;
+
+  afterEach(async () => {
+    for (const backend of backendInstances) {
```

Comment:
`Promise.all`? 😄 Not that they're expected to take long, but every little bit matters i guess.